app/src/main/res/layout/fragment_home.xml

```diff
@@ -16,4 +17,33 @@
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
+
+    <EditText
+        android:id="@+id/txtDate"
```

Comment:
thanks for renaming this textbox!

Reply:
I think txtDate is appropriate name as we need to retrieve standup notes by date /day.